src/Modal/index.jsx

```diff
@@ -20,12 +26,21 @@ class Modal extends React.Component {
 
     this.headerId = newId();
 
+    this.el = document.createElement('div');
+
+    if (!modalRoot) {
+      modalRoot = document.createElement('div');
+      modalRoot.id = modalRootId;
+      document.body.appendChild(modalRoot);
```

Comment:
What happens if I want to append this modal to a specific parent in the DOM? Should we add a `parentSelector` prop that defaults to the document body?

Reply:
We could add a parentSelector prop, though I believe we do expect to want to always render the modal at the top level component, that was at a suggestion from @arizzitano who may have more thoughts on that.